src/plugins/sails.js

```diff
@@ -75,6 +75,7 @@ io.socket.on('game', function(evData) {
 				}
 			}
 			break;
+		case 'targetedOneOff':
```

Comment:
I don't understand how we were able to play these before without this! Did the countering dialog pop up previously?

Reply:
I noticed this when writing a test case from opponent's perspective. previously with targeted one-offs the test cases were only from player's perspective and the game probably didn't need to check if there was a coutering dialog or not.